cross/psutil/Makefile

```diff
@@ -0,0 +1,12 @@
+PKG_NAME = psutil
```

Comment:
I had to introduce the cross compiled version of psutil.
Indeed there was an issue which made the pip process fail building the corresponding wheel.